configuration/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### Unreleased
 
+- feature: add more explicit Processor Config TS declaration
+- refactor: make Processor config keys optional, and prevent trivial ser.
```

Comment:
I think this was truncated?

Reply:
oh I mean serialization. changed it to be more readable